README.md

```diff
@@ -8,28 +7,67 @@ Instabug is a bug reporting and in-app feedback tool that provides your testers
 For more info, visit [Instabug.com](https://instabug.com).
 
 ### Installation
+
 Currently, this plugin can only be installed via the Command-Line Interface.
+
 ```
 cordova plugin add instabug-cordova
 ```
+
 ##### Ionic
+
 If you're using this plugin with Ionic, you can install using this command.
+
 ```
 ionic cordova plugin add instabug-cordova
 ```
 
 ## Usage
-1. You can initialize the SDK by using this method in your JS class
+
+To initialize Instabug in your app, you need to do the following:
+
+### Android
+
+1. Change the name of the application class in your AndroidManifest.xml file to `android:name="com.instabug.cordova.plugin.MyApplication"`.
+
+Since `Cordova Android 7.0.0` AndroidManifest.xml position has changed and it is placed under `app/src/main/AndroidManifest.xml` ([read more](http://cordova.apache.org/announcements/2017/12/04/cordova-android-7.0.0.html)). You have to point it in your `.gradle` file in `platforms\android\com.instabug.cordova.plugin` directory in this way:
+
+```
+sourceSets {
+        main {
+            manifest.srcFile 'src/main/AndroidManifest.xml'
+        }
+    }
+```
+
+2. You need to add your app token in the **MyApplication** class, by replacing `YOUR_ANDROID_TOKEN`. (You can find this class under this path `YourProjectName/platforms/android/app/src/main/java/com.instabug.cordova.plugin/MyApplication.java`)
```

Comment:
It seems the path has been changed to `plugins/instabug-cordova/src/android/MyApplication.java` and therefore no integration code was added at my end.

Reply:
You will find it in the path mentioned in the Readme too.